src/main/java/command/Rectangle.java

```diff
@@ -0,0 +1,30 @@
+package command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class Rectangle implements ActionListener {
```

Comment:
It is not a factory. There is no method returning DriverCommand. Factories should be extracted outside listeners.